modules/related-posts/jetpack-related-posts.php

```diff
@@ -172,11 +172,13 @@ public function get_headline() {
 	 * @returns string
 	 */
 	public function filter_add_target_to_dom( $content ) {
+		global $wp_current_filter;
+
 		if ( has_block( 'jetpack/related-posts', $content ) ) {
 			return $content;
 		}
 
-		if ( ! $this->_found_shortcode ) {
+		if ( ! $this->_found_shortcode && ! in_array( 'get_the_excerpt', (array) $wp_current_filter ) ) {
```

Comment:
Better to use `doing_filter( 'get_the_excerpt' )` rather than access the global directly.

Reply:
Thanks! The newest commit uses `doing_filter()`.